From: Andrew Morton <akpm@linux-foundation.org>
Date: Tue, 11 Aug 2026 18:18:29 -0700
Subject: [PATCH] 
 mm-provide-vma_is_cow_mapping-and-remove-is_cow_mapping-fix-fix

fix kerneldoc

Cc: "Lorenzo Stoakes (ARM)" <ljs@kernel.org>
Reported-by: Randy Dunlap <rdunlap@infradead.org>
Closes: https://lore.kernel.org/e7911812-9b99-4f94-be6b-597bf202ffeb@infradead.org
Tested-by: Randy Dunlap <rdunlap@infradead.org>
Acked-by: Randy Dunlap <rdunlap@infradead.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 include/linux/mm.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/mm.h b/include/linux/mm.h
index 17ae65f227107..c97d4f76267d4 100644
--- a/include/linux/mm.h
+++ b/include/linux/mm.h
@@ -2319,7 +2319,7 @@ static inline bool vma_flags_is_cow_mapping(const vma_flags_t *flags)
 
 /**
  * vma_is_cow_mapping() - Is this VMA a CoW mapping?
- * @desc: The VMA to check.
+ * @vma: The VMA to check.
  *
  * See vma_flags_is_cow_mapping() for details.
  *
-- 
2.47.3

